### PR TITLE
Release Schedule

### PR DESCRIPTION
**Pull request by dns2utf8**
- [x] times on all talks
- [x] workshops
    - [ ] times do we start 10:00?
    - [x] locations did we already assign them? Or do that later?
- [x] undo the path changes of the talk session

bildschirmfoto am 2018-10-30 um 21 24 45

### CODE REVIEW

**Comment by gandro** [comment]
I don't think we have to assign rooms right now. Some hosts might have room requirements I don't know about yet - let's publish this as it without the rooms.

**Comment by dns2utf8** [comment]
Okay, will remove the times for now

**Comment by gandro** [comment]
The times for the workshops are fine with me, we can still change them. But the important thing is that links don't break. Otherwise looks good to me!

File: _sessions/parallel-programming.md [hunk 1]
```diff
@@ -8,7 +8,10 @@ speakers:
 
 title: Parallel Programming
 desc: >
-  Multiprocessor programming is a big mystery to many people. This is an introduction workshop for beginners explaining different modes on execution and then use channels to develop a multi-plattform, parallel thumbnail generator. Using ThreadPool and rayon.
+  Multiprocessor programming is a big mystery to many people. This is an introduction workshop for beginners explaining different modes on execution and then use channels to develop a multi-plattform, parallel thumbnail generator. 
+
+
+  Using the crates ThreadPool and rayon we will learn how easy it is to parallelize workloads with rust.
```
Comment (apiraino) [inline _sessions/parallel-programming.md:14]:
```suggestion
  Using the crates [ThreadPool](https://crates.io/crates/threadpool) and [rayon](https://crates.io/crates/rayon) we will learn how easy it is to parallelize workloads with rust.
```

File: _sessions/parallel-programming.md [hunk 2]
```diff
@@ -8,7 +8,10 @@ speakers:
 
 title: Parallel Programming
 desc: >
-  Multiprocessor programming is a big mystery to many people. This is an introduction workshop for beginners explaining different modes on execution and then use channels to develop a multi-plattform, parallel thumbnail generator. Using ThreadPool and rayon.
+  Multiprocessor programming is a big mystery to many people. This is an introduction workshop for beginners explaining different modes on execution and then use channels to develop a multi-plattform, parallel thumbnail generator.
```
Comment (apiraino) [inline _sessions/parallel-programming.md:11]:
```suggestion
  Multiprocessor programming is a big mystery to many people. This is an introduction workshop for beginners explaining different modes on execution and then use channels to develop a multi-platform, parallel thumbnail generator. 
```

**Review by apiraino — COMMENTED** [review]
looks good to me!

**Comment by gandro** [comment]
I took the liberty of undoing the session rename so this can be published in the morning! Feel free to revert if necessary!

File: _sessions/parallel-programming.md (hunk 1 above)
Comment (dns2utf8) [reply _sessions/parallel-programming.md:14]: Thanks, applied it directly.

**Pull request merged**